site/en/docs/webstore/manage/index.md

```diff
@@ -119,9 +119,10 @@ Users will be able to communicate with you using the **Support** tab of your sto
 
 ### Using the User Support tab {: #user-support-tab }
 
-First, you must enable user feedback in the developer console Account setting page.
+Having a Support section on your extension's Chrome Web Store page allows users to easily report issues and suggest new features separately from leaving a review. To enable user feedback on your extension, the Support Visibility option must be switched on.   
+The Support Visibility option switch is located in the Chrome Web Store Developer Dashboard, in the Support Tab section of Account Options.
 
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/ArMfJztL1OlP6UektUwb.png", alt="Enable User Feedback Support tab", width="800", height="185" %}
+{% Img src="image/6AZNJBRnkpQUWTKPzig99lQY8jT2/3h8A6IglCHjMN4uncopz.png", alt="Support Visibility option", width="800", height="239" %}
```

Comment:
```suggestion
{% Img src="image/6AZNJBRnkpQUWTKPzig99lQY8jT2/3h8A6IglCHjMN4uncopz.png", alt="Visibility option", width="800", height="239" %}
```

Reply:
Please don't use screen captures that are unreadable. Cutting off the help doesn't hurt at all. I'd suggest something like this. Feel free to use it. 

![SupportTab](https://user-images.githubusercontent.com/8204171/200418358-5086328c-4c35-4b17-8537-baf4097911ee.png)